client/styles/components/_console.scss

```diff
@@ -92,12 +92,22 @@
 	}
 	background: transparent;
 	border: none;
+	border-radius: 20px;
+	margin-right: 5px; 
 	padding-right: #{10 / $base-font-size}rem;
 	.preview-console--collapsed & {
 		display: none;
 	}
 }
 
+.preview-console__clear:hover {
+
+	background-color: rgba(45, 38, 38, 0.345);
+	border-radius: 20px;
```

Comment:
You don't need this here since you already added a `border-radius` to element via line 95.